pyvelox/pyvelox.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+#include <velox/type/Type.h>
+
+namespace facebook::pyvelox {
```

Comment:
Wondering is the namespace hierarchy here should be something like `facebook::velox::py`, since the pyvelox stuff is also part of velox. 

I'm thinking that in the future we might have things like `facebook::velox::java` for jni bindings and such.

Reply:
Sounds good - will make that change.